CMakeLists.txt

```diff
@@ -195,6 +195,24 @@ if(WITH_GSL)
   endif(GSL_FOUND)
 endif(WITH_GSL)
 
+find_package(BLAS)
+if(BLAS_FOUND)
+  set(BLAS 1)
+endif()
+find_package(LAPACK)
+if(LAPACK_FOUND)
+  set(LAPACK 1)
+endif()
+
+if(BLAS AND LAPACK)
+  set(STOKESIAN_DYNAMICS 1)
+endif()
+
+# Temporarily disabled to test CPU on CI
+if(WITH_CUDA)
```

Comment:
Can't you steer that from the command line? ```-DSTOKESIAN_DYNAMICS_GPU=1``` or similar

Reply:
That looks like a testing leftover.

Reply:
not as long as I want it to be tested that way on CI afaik

Reply:
Th GPU version hasn't been tested yet. Depends on cuBLAS to be avialable on the respective images.

Reply:
Sorry, should have read the entire code. Just remove the `# Temporarily disabled to test CPU on CI` comment.

Lines 212-214 are essentially equivalent to lines 207-209, just for GPU instead of CPU. So they can stay the way they are.